src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/TagHelperAttribute.cs

```diff
@@ -8,77 +8,70 @@ namespace Microsoft.AspNetCore.Razor.TagHelpers
     /// <summary>
     /// An HTML tag helper attribute.
     /// </summary>
-    public class TagHelperAttribute : IReadOnlyTagHelperAttribute
+    public class TagHelperAttribute
     {
         private static readonly int TypeHashCode = typeof(TagHelperAttribute).GetHashCode();
 
         /// <summary>
-        /// Instantiates a new instance of <see cref="TagHelperAttribute"/>.
+        /// Instantiates a new instance of <see cref="TagHelperAttribute"/> with the specified <paramref name="name"/>.
+        /// <see cref="Minimized"/> is set to <c>true</c> and <see cref="Value"/> to <c>null</c>.
         /// </summary>
-        public TagHelperAttribute()
+        /// <param name="name">The <see cref="Name"/> of the attribute.</param>
+        public TagHelperAttribute(string name)
+            : this(name, value: null, minimized: true)
         {
         }
 
         /// <summary>
-        /// Instantiates a new instance of <see cref="TagHelperAttribute" /> with values provided by the given
-        /// <paramref name="attribute"/>.
+        /// Instantiates a new instance of <see cref="TagHelperAttribute"/> with the specified <paramref name="name"/>
+        /// and <paramref name="value"/>. <see cref="Minimized"/> is set to <c>false</c>.
         /// </summary>
-        /// <param name="attribute">A <see cref="IReadOnlyTagHelperAttribute"/> whose values should be copied.</param>
-        public TagHelperAttribute(IReadOnlyTagHelperAttribute attribute)
```

Comment:
A `TagHelperAttribute(IReadOnlyTagHelperAttribute attribute, object value)` overload would be useful though it should probably `throw` something if `attribute.Minimized`.